modules/ossm-enabling-sidecar-injection.adoc

```diff
@@ -0,0 +1,141 @@
+// Module included in the following assemblies:
+// install/ossm-sidecar-injection
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-enabling-sidecar-injection_{context}"]
+= Enabling sidecar injection
+
+The sidecar injection process in {istio} uses label-based rules to determine if a pod should be injected with a sidecar. If a namespace has both the `istio-injection` label and the `istio.io/rev` label, the `istio-injection` label takes precedence, treating the namespace as part of the default revision.
+
+The sidecar injector follows the following logic:
+
+* If either the `istio-injection` label or the `sidecar.istio.io/inject` label is set to disabled, the pod is not injected.
+* If any of the following labels are enabled the pod is injected with a sidecar: `istio-injection`, `sidecar.istio.io/inject`, or `istio.io/rev`.
```

Comment:
Perhaps start with the fact that all pods are injected by default (if injection is enabled at the namespace level). Then explain how to use the `istio-injection: disabled` label to disable injection for particular pods.

Reply:
@luksa Rephrased the content. PTAL. Thanks!